
---------------
 /home/marcelo/repos/MAPpoly/man/est_pairwise_rf.Rd  --  2020-09-23 16:43:02 

seq.ch1 <- make_seq_mappoly(mrks.chi.filt) 
- Error in get(input.obj$data.name, pos = 1): object 'dat.filt.ind' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/est_pairwise_rf.Rd  --  2020-09-23 16:43:02 

plot(seq.ch1) 
- Error in plot(seq.ch1): object 'seq.ch1' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/est_pairwise_rf.Rd  --  2020-09-23 16:43:02 

all.pairs.1 <- est_pairwise_rf(input.seq = seq.ch1, ncpus = 7,      verbose = TRUE) 
- Error in est_pairwise_rf(input.seq = seq.ch1, ncpus = 7, verbose = TRUE): object 'seq.ch1' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/est_pairwise_rf.Rd  --  2020-09-23 16:43:02 

all.pairs.2 <- est_pairwise_rf(input.seq = seq.ch1, ncpus = 7,      n.batch = 10, verbose = TRUE) 
- Error in est_pairwise_rf(input.seq = seq.ch1, ncpus = 7, n.batch = 10, : object 'seq.ch1' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/est_pairwise_rf.Rd  --  2020-09-23 16:43:02 

plot(all.pairs.1, 161, 162) 
- Error in plot(all.pairs.1, 161, 162): object 'all.pairs.1' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/est_pairwise_rf.Rd  --  2020-09-23 16:43:02 

mat <- rf_list_to_matrix(all.pairs.1) 
- Error in rf_list_to_matrix(all.pairs.1): object 'all.pairs.1' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_from_updog.Rd  --  2020-09-23 17:14:33 

mydata = import_from_updog(mout, filter.non.conforming = TRUE) 
- Error in dimnames(M) <- list(input.data$mrk.names, 0:m): length of 'dimnames' [2] not equal to array extent
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:38 

genoprob0 <- lapply(mappoly.maplist, calc_genoprob, step = 1) 
- Error in get(input.map$info$data.name, pos = 1): object 'mappoly.data' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:38 

pref.pair0 <- calc_prefpair_profiles(genoprob0) 
- Error in calc_prefpair_profiles(genoprob0): object 'genoprob0' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:38 

plot(pref.pair0, min.y.prof = 0.25, max.y.prof = 0.4, P = "P1",      Q = "P2") 
- Error in plot(pref.pair0, min.y.prof = 0.25, max.y.prof = 0.4, P = "P1", : object 'pref.pair0' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:38 

h.prob0 <- calc_homoprob(genoprob0) 
- Error in calc_homoprob(genoprob0): object 'genoprob0' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:38 

plot(h.prob0, ind = "F1_030") 
- Error in plot(h.prob0, ind = "F1_030"): object 'h.prob0' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:38 

genoprob1 <- lapply(mappoly.maplist, calc_genoprob_error, step = 1,      error = 0.05) 
- Error in get(input.map$info$data.name, pos = 1): object 'mappoly.data' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:38 

pref.pair1 <- calc_prefpair_profiles(genoprob1) 
- Error in calc_prefpair_profiles(genoprob1): object 'genoprob1' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:38 

plot(pref.pair1, min.y.prof = 0.25, max.y.prof = 0.4, P = "P1",      Q = "P2") 
- Error in plot(pref.pair1, min.y.prof = 0.25, max.y.prof = 0.4, P = "P1", : object 'pref.pair1' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:38 

h.prob1 <- calc_homoprob(genoprob1) 
- Error in calc_homoprob(genoprob1): object 'genoprob1' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:38 

plot(h.prob1, ind = "F1_030") 
- Error in plot(h.prob1, ind = "F1_030"): object 'h.prob1' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:47 

parallel::clusterExport(cl, "mappoly.data") 
- Error in get(name, envir = envir): object 'mappoly.data' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:47 

reest.maps <- parallel::parLapply(cl, mappoly.maplist, est_full_hmm_with_global_error,      error = 0.05) 
- Error in checkForRemoteErrors(val): 5 nodes produced errors; first error: object 'mappoly.data' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:47 

genoprob2 <- lapply(reest.maps, calc_genoprob_error, step = 1,      error = 0.05) 
- Error in lapply(reest.maps, calc_genoprob_error, step = 1, error = 0.05): object 'reest.maps' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:47 

pref.pair2 <- calc_prefpair_profiles(genoprob2) 
- Error in calc_prefpair_profiles(genoprob2): object 'genoprob2' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:47 

plot(pref.pair2, min.y.prof = 0.25, max.y.prof = 0.4, P = "P1",      Q = "P2") 
- Error in plot(pref.pair2, min.y.prof = 0.25, max.y.prof = 0.4, P = "P1", : object 'pref.pair2' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:47 

h.prob2 <- calc_homoprob(genoprob2) 
- Error in calc_homoprob(genoprob2): object 'genoprob2' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:47 

plot(h.prob2, ind = "F1_030") 
- Error in plot(h.prob2, ind = "F1_030"): object 'h.prob2' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:47 

tpt <- est_pairwise_rf(input.seq = s, ncpus = 7) 
- Error in get(input.seq$data.name, pos = 1): object 'mappoly.data' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:47 

mat <- rf_list_to_matrix(make_pairs_mappoly(tpt, s)) 
- Error in make_pairs_mappoly(tpt, s): object 'tpt' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:47 

grs <- group_mappoly(input.mat = mat, expected.groups = 5, inter = TRUE) 
- Error in group_mappoly(input.mat = mat, expected.groups = 5, inter = TRUE): object 'mat' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:47 

grs 
- Error in eval(rd_ex_parsed[[repi]], envir = env): object 'grs' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:47 

for (i in 1:5) {     s.temp <- make_seq_mappoly(grs, arg = i)     tpt.temp <- make_pairs_mappoly(tpt, s.temp)     sf <- rf_snp_filter(input.twopt = tpt.temp, thresh.LOD.ph = 1,          thresh.LOD.rf = 1, thresh.perc = 0.02)     M <- make_mat_mappoly(input.mat = mat, sf)     o <- mds_mappoly(M)     so <- make_seq_mappoly(o)     plot(M, ord = so$seq.mrk.names, main.text = paste("LG", i),          index = FALSE)     LG[[i]] <- list(s = so, tpt = tpt.temp)     cat("\n") } 
- Error in make_seq_mappoly(grs, arg = i): object 'grs' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:47 

for (i in 1:5) {     MAPs[[i]] <- est_rf_hmm_sequential(input.seq = LG[[i]]$s,          start.set = 6, thres.twopt = 10, thres.hmm = 50, extend.tail = 30,          twopt = LG[[i]]$tpt, verbose = TRUE, tol = 0.1, tol.final = 0.001,          phase.number.limit = 20, sub.map.size.diff.limit = 5,          info.tail = TRUE, reestimate.single.ph.configuration = TRUE) } 
- Error in est_rf_hmm_sequential(input.seq = LG[[i]]$s, start.set = 6, thres.twopt = 10, : LG[[i]]$s is not an object of class 'mappoly.sequence'
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:53 

parallel::clusterExport(cl, "mappoly.data") 
- Error in get(name, envir = envir): object 'mappoly.data' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:53 

recons.maps <- parallel::parLapply(cl, MAPs, est_full_hmm_with_global_error,      error = 0.05) 
- Error in checkForRemoteErrors(val): 5 nodes produced errors; first error: X[[i]] is not an object of class 'mappoly.map'
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:53 

summary_maps(mappoly.maplist) 
- Error in get(x$info$data.name, pos = 1): object 'mappoly.data' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:53 

summary_maps(recons.maps) 
- Error in lapply(X = X, FUN = FUN, ...): object 'recons.maps' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:53 

genoprob3 <- lapply(recons.maps, calc_genoprob_error, step = 1,      error = 0.05) 
- Error in lapply(recons.maps, calc_genoprob_error, step = 1, error = 0.05): object 'recons.maps' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:53 

pref.pair3 <- calc_prefpair_profiles(genoprob3) 
- Error in calc_prefpair_profiles(genoprob3): object 'genoprob3' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:53 

plot(pref.pair3, min.y.prof = 0.25, max.y.prof = 0.4, P = "P1",      Q = "P2") 
- Error in plot(pref.pair3, min.y.prof = 0.25, max.y.prof = 0.4, P = "P1", : object 'pref.pair3' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:53 

h.prob3 <- calc_homoprob(genoprob3) 
- Error in calc_homoprob(genoprob3): object 'genoprob3' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:53 

plot(h.prob0, ind = "F1_030", use.plotly = FALSE) 
- Error in plot(h.prob0, ind = "F1_030", use.plotly = FALSE): object 'h.prob0' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:53 

plot(h.prob1, ind = "F1_030", use.plotly = FALSE) 
- Error in plot(h.prob1, ind = "F1_030", use.plotly = FALSE): object 'h.prob1' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:53 

plot(h.prob2, ind = "F1_030", use.plotly = FALSE) 
- Error in plot(h.prob2, ind = "F1_030", use.plotly = FALSE): object 'h.prob2' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/import_phased_maplist_from_polymapR.Rd  --  2020-09-23 17:14:53 

plot(h.prob3, ind = "F1_030", use.plotly = FALSE) 
- Error in plot(h.prob3, ind = "F1_030", use.plotly = FALSE): object 'h.prob3' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/make_seq_mappoly.Rd  --  2020-09-23 17:53:18 

seq1 = make_seq_mappoly(grs, arg = 1, genomic.info = 1) 
- Error in -c(ncol(input.obj$seq.vs.grouped.snp)): invalid argument to unary operator
 

---------------
 /home/marcelo/repos/MAPpoly/man/merge_datasets.Rd  --  2020-09-23 17:56:57 

seq.init <- make_seq_mappoly(mrks.chi.filt) 
- Error in get(input.obj$data.name, pos = 1): object 'dat' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/merge_datasets.Rd  --  2020-09-23 17:56:57 

length(seq.init$seq.mrk.names) 
- Error in eval(rd_ex_parsed[[repi]], envir = env): object 'seq.init' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/merge_datasets.Rd  --  2020-09-23 17:56:57 

plot(seq.init) 
- Error in plot(seq.init): object 'seq.init' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/merge_datasets.Rd  --  2020-09-23 17:56:57 

print(seq.init, detailed = TRUE) 
- Error in print(seq.init, detailed = TRUE): object 'seq.init' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/read_vcf.Rd  --  2020-09-23 18:15:35 

seq.init <- make_seq_mappoly(mrks.chi.filt) 
- Error in get(input.obj$data.name, pos = 1): object 'dat' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/read_vcf.Rd  --  2020-09-23 18:15:35 

length(seq.init$seq.mrk.names) 
- Error in eval(rd_ex_parsed[[repi]], envir = env): object 'seq.init' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/read_vcf.Rd  --  2020-09-23 18:15:35 

plot(seq.init) 
- Error in plot(seq.init): object 'seq.init' not found
 

---------------
 /home/marcelo/repos/MAPpoly/man/read_vcf.Rd  --  2020-09-23 18:15:35 

print(seq.init, detailed = TRUE) 
- Error in print(seq.init, detailed = TRUE): object 'seq.init' not found
 
